packages/grid/_modules_/grid/components/column-header-filter-icon.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import { useIcons } from '../hooks/utils/useIcons';
+import { COLUMN_FILTER_BUTTON_CLICK } from '../constants';
+import { ApiContext } from './api-context';
+import { ColDef } from '../models/colDef/colDef';
+import {
+  Box,
+  ClickAwayListener,
+  Paper,
+  Popper,
+  Tab,
+  Tabs,
+  TextField,
+  Typography,
+} from '@material-ui/core';
```

Comment:
I think that we should use 1-level deep import systematically. If you don't, you load the whole `@material-ui/core` package in development, which slows down the DX. 

Speaking of DX https://deploy-preview-380--material-ui-x.netlify.app/storybook/ takes 23s for me to load the assets from the network (at least from the Airbnb's Wi-Fi) and to display the first page.

<img width="149" alt="Capture d’écran 2020-10-01 à 00 34 32" src="https://user-images.githubusercontent.com/3165635/94746843-eacc8780-037d-11eb-9505-adaa924ad9ae.png">